### ISSUE TITLE
Form group on the “My Health Log Symptoms List” screen is not programmatically associated with its group label [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#299]
The “Do you have any of the following 11 symptoms today?” form group label is not announced upon focusing a control in the form group. For example, the “Muscles aches or body aches” checkbox is announced as “Muscle aches or body aches, unselected, Muscle aches or body aches, unselected, checkbox”.

![My Health Log - NRV - Checkbox unselected](https://user-images.githubusercontent.com/7034772/92132150-c9a36600-edd4-11ea-81ab-c65342bf61db.png)

Use the `accessibilityLabel` property to include the group label in the accessible name of each control. Remove the group label from the reading order using the `accessibilityElementsHidden`, as it is redundant when included in the `accessibilityLabel` of each checkbox.

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Do you have any of the following 11 symptoms today?</Text>
</View>

<TouchableOpacity
  onPress={…}
  accessibilityLabel='Muscle aches or body aches, Do you have any of the following 11 symptoms today?'
  accessibilityRole='checkbox'
  accessibilityState={{selected: true}}
>
```

**Comment by floridemai** [comment]
@jonnyjames This should be fixed in the next build for re-testing.

**Comment by jonnyjames** [comment]
@floridemai This is now fixed. Closing this issue.
